### ISSUE TITLE
error: /bin/sh: 1: conda: not found 

### ISSUE THREAD

**Issue by hailianghuang** [stephenslab/susie-paper#7]
When running 'docker build -t susie/susie-paper .', I got this error. The error happened on both Mac and Ubuntu.
 
```
Sending build context to Docker daemon  20.17MB
Step 1/31 : FROM debian:stretch-slim
 ---> 942afde0bfe6
Step 2/31 : MAINTAINER Gao Wang, gaow@uchicago.edu
 ---> Using cache
 ---> 7c6b810f05e4
Step 3/31 : WORKDIR /tmp
 ---> Using cache
 ---> 76b3b6f4c7f7
Step 4/31 : RUN apt-get update     && apt-get install -y --no-install-recommends     curl     unzip     gzip     bzip2     ca-certificates     build-essential     gfortran     libgfortran-6-dev     libgomp1     && apt-get clean     && rm -rf /var/lib/apt/lists/* /var/log/dpkg.log
 ---> Using cache
 ---> 833ae5cf0109
Step 5/31 : RUN apt-get update     && apt-get install -y --no-install-recommends     libgsl-dev libgsl2 libatlas3-base liblapack-dev     && apt-get clean
 ---> Using cache
 ---> 94c62ecba893
Step 6/31 : ENV MINICONDA_VERSION 4.7.12
 ---> Using cache
 ---> 1bfe67b7a164
Step 7/31 : ENV PATH /opt/miniconda3/bin:$PATH
 ---> Using cache
 ---> a275715e5208
Step 8/31 : RUN curl https://repo.continuum.io/miniconda/Miniconda3-$MINICONDA_VERSION-Linux-x86_64.sh -o MCON.sh     && /bin/bash MCON.sh -b -p /opt/miniconda3     && ln -s /opt/miniconda3/etc/profile.d/conda.sh /etc/profile.d/conda.sh     && conda install -c conda-forge r-base==3.5.1 rpy2==3.1.0 r-reticulate==1.13 			matplotlib==3.0.2 seaborn==0.9.0     && conda clean --all -tipsy && rm -rf /tmp/* $HOME/.cache
 ---> Running in 6c20808c2283
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
/bin/sh: 1: conda: not found
The command '/bin/sh -c curl https://repo.continuum.io/miniconda/Miniconda3-$MINICONDA_VERSION-Linux-x86_64.sh -o MCON.sh     && /bin/bash MCON.sh -b -p /opt/miniconda3     && ln -s /opt/miniconda3/etc/profile.d/conda.sh /etc/profile.d/conda.sh     && conda install -c conda-forge r-base==3.5.1 rpy2==3.1.0 r-reticulate==1.13 			matplotlib==3.0.2 seaborn==0.9.0     && conda clean --all -tipsy && rm -rf /tmp/* $HOME/.cache' returned a non-zero code: 127

```

**Comment by gaow** [comment]
@hailianghuang It turns out that unfortunately (and a bit surprisingly) `miniconda` changed their URL to download archived packages so `conda` never got installed ! I just looked it up and replaced the URL with the current one, and pushed it to `master`.

I would also like to note that the Docker image was meant for archiving: it provides an easy way to set up an exact environment to analyse the data associated with the SuSiE paper publication, and is self-documenting. It is intended for the reproducibility of results in the manuscript, but many updates have been implemented to `susieR` since the manuscript was written. If your goal is to analyze data, our recommendation is to install the current stable version of susieR, on the `master` branch on our github:

```
devtools::install_github('stephenslab/susieR')
```
